presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHadoopAuthentication.java

```diff
@@ -38,10 +41,13 @@ public static KerberosHadoopAuthentication createKerberosHadoopAuthentication(Ke
         // UserGroupInformation.authenticationMethod static field must be set to KERBEROS
         // It is further used in many places in DfsClient
         configuration.set("hadoop.security.authentication", "kerberos");
-        String authToLocalRules = configuration.get("hadoop.security.auth_to_local");
-        if (authToLocalRules == null) {
-            // KerberosName#rules static field must be initialized if hadoop.security.auth_to_local is null
-            KerberosName.setRules("DEFAULT");
+
+        // Set the static configuration to get the rules.
+        try {
+            HadoopKerberosName.setConfiguration(configuration);
```

Comment:
It is being calles inside the `UserGroupInformation.setConfiguration`. Please remove that. https://github.com/prestodb/presto/pull/11380#discussion_r214222456